content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -345,11 +345,16 @@ The control plane adds this label to an Endpoints object when the owning Service
 
 ### kubernetes.io/service-name {#kubernetesioservice-name}
 
-Example: `kubernetes.io/service-name: "nginx"`
+Example: `kubernetes.io/service-name: "my-website"`
 
-Used on: Service
+Used on: EndpointSlice
+
+Kubernetes associates [EndpointSlices](/docs/concepts/services-networking/endpoint-slices/) with
+[Services](/docs/concepts/services-networking/service/) using this label.
 
-Kubernetes uses this label to differentiate multiple Services. Used currently for `ELB`(Elastic Load Balancer) only.
+This label records the {{< glossary_tooltip term_id="name" text="name">}} of the
+Service that the EndpointSlice is backing. All EndpointSlices should have this label set to
+the name of their associated Service.
```

Comment:
Is this correct?

Reply:
From the [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/0752-endpointslices)
> type EndpointSlice struct {
>     metav1.TypeMeta `json:",inline"`
>     // OwnerReferences should be set when the object is derived from a k8s
>     // object.
>     // The object labels may include the following keys:
>     // * kubernetes.io/service-name: the label value indicates the name of the
>     //   service from which the EndpointSlice is derived.

I think the PR is correct but is it more appropriate to point out that the EndpointSlice is derived from the Service of of the label's value.

Reply:
I also notice another label from the [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/0752-endpointslices)

>  // * endpointslice.kubernetes.io/managed-by: the label value represents a
>     //   unique name for the controller or application that manages this
>     //   EndpointSlice.

Reply:
> I think the PR is correct but is it more appropriate to point out that the EndpointSlice is derived from the Service of of the label's value.

Technically, you could have an EndpointSlice, and apply an update to change which Service it references (maybe for some canary releasing thing?). That imaginary EndpointSlice isn't _derived_ from either Service.

I won't add `endpointslice.kubernetes.io/managed-by` in this PR unless folks insist on it. This PR is already too big.

Reply:
@robscott is the best person to judge this